text/0181-deprecate-ember-data-initializers.md

```diff
@@ -0,0 +1,68 @@
+- Start Date: 2016-11-22
+- RFC PR: (leave this empty)
+- Ember Issue: (leave this empty)
+
+# Summary
+
+The goal of this RFC is to remove the `data-adapter`, `injectStore`,
+`transforms`, and `store` Ember initializers that Ember Data injects
```

Comment:
possibly better to refer to them as application/application instance initializers?